TwilioStore.Services/OrderService.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Configuration;
```

Comment:
Turns out that `System.Configuration` and `Twilio` are not required in this class. _Disclaimer: I cheated with Visual Studio_.